sphinx/source/docs/dev_guide/server.rst

```diff
@@ -258,8 +259,8 @@ two sequences of frames, one sequence in each direction ("full duplex").
 On top of websocket frames, we implement our own ``Message`` concept. A Bokeh
 ``Message`` spans multiple websocket frames. It always contains a header frame,
 metadata frame, and content frame. These three frames each contain a JSON
-string. The code permits these three frames to be followed by binary data
-frames, but currently in Bokeh binary data frames are not used.
+string. The code permits these three frames to be followed by optional binary data
+frames. Currently in Bokeh, binary data frames are only used to transport image data.
```

Comment:
It's a small list, we should be explicit: all NumPy arrays with `float32`, `float64`, and `int32` dtypes, this is correct?